app/dao/organisation_permissions_dao.py

```diff
@@ -0,0 +1,42 @@
+from app import db
+from app.dao import DAOClass
+from app.models import OrganisationPermission
+
+# Default permissions for a service
+default_organisation_permissions = []
+
+
+class OrganisationPermissionDAO(DAOClass):
+    class Meta:
+        model = OrganisationPermission
+
+    def add_default_service_permissions_for_organisation(self, organisation):
+        for name in default_organisation_permissions:
+            organisation_permission = OrganisationPermission(permission=name, organisation=organisation)
+            self.create_instance(organisation_permission, _commit=False)
+
+    def remove_user_service_permissions(self, organisation):
+        query = self.Meta.model.query.filter_by(organisation=organisation)
+        query.delete()
```

Comment:
I think we can remove references to `service` here - ie just call them `add_default_permissions_to_organisation` and `remove_organisation_permissions`

Reply:
I always leave behind stragglers like this... not to worry.